integration/test/app/workload/workload_cluster_test.go

```diff
@@ -61,6 +71,21 @@ func TestWorkloadCluster(t *testing.T) {
 			t.Fatalf("expected nil got %#v", err)
 		}
 
+		// Create kubeconfig secret for the chart CR watcher in app-operator.
+		secret := &corev1.Secret{
+			Data: map[string][]byte{
+				"kubeConfig": bytes,
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("%s-kubeconfig", key.WorkloadClusterNamespace()),
+				Namespace: key.WorkloadClusterNamespace(),
+			},
+		}
+		_, err = config.K8sClients.K8sClient().CoreV1().Secrets(key.WorkloadClusterNamespace()).Create(ctx, secret, metav1.CreateOptions{})
+		if err != nil {
+			t.Fatalf("expected nil got %#v", err)
+		}
```

Comment:
@tomahawk28 Good catch on the errors in the logs. 

I extended the test to create this kubeconfig secret and now the chart CR watcher works for both app-operators. PTAL